alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -1045,4 +1082,25 @@ class MonitorRestApiIT : AlertingRestTestCase() {
             assertEquals("Unexpected status", RestStatus.BAD_REQUEST, e.response.restStatus())
         }
     }
+
+    private fun validateAlertingStatsNodeResponse(nodesResponse: Map<String, Int>) {
+        assertEquals("Incorrect number of nodes", numberOfNodes, nodesResponse["total"])
+        assertEquals("Failed nodes found during monitor stats call", 0, nodesResponse["failed"])
+        assertEquals("More than $numberOfNodes successful node", numberOfNodes, nodesResponse["successful"])
+    }
+
+    private fun isMonitorScheduled(monitorId: String, alertingStatsResponse: Map<String, Any>): Boolean {
+        var monitorScheduled = false
+        val nodesInfo = alertingStatsResponse["nodes"] as Map<String, Any>
+        for (nodeId in nodesInfo.keys) {
+            val nodeInfo = nodesInfo[nodeId] as Map<String, Any>
+            val jobsInfo = nodeInfo["jobs_info"] as Map<String, Any>
+            if (jobsInfo.keys.contains(monitorId)) {
+                monitorScheduled = true
+                break
```

Comment:
Why not just return true here and on the bottom of the function, have it return false?

Reply:
Good point, I think there was an assertion in the method originally which is why it was saved as a variable.